release/promote.sh

```diff
@@ -70,16 +71,46 @@ promote_pypi() {
     echo
 }
 
+# Promote dependencies
+promote_s3 "certifi" whl "2022.12.7"
```

Comment:
This is not my favorite thing in the world but security right?

Reply:
Yes will have to add it to runbook if new versions of dependencies are uploaded we should do the change here